### ISSUE TITLE
Contribute: Add proper header for export course download

### ISSUE THREAD

**Issue by OmarIthawi** [appsembler/edx-platform#560]
https://github.com/appsembler/edx-platform/pull/559 seems like a good fix to contribute upstream.

**Comment by thraxil** [comment]
Agreed, although it would have to be part of a larger patch to add `boto3` support since they're still using the old version: https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/import_export.py#L376

**Comment by OmarIthawi** [comment]
Thanks Anders for the info.
